pelican/contents.py

```diff
@@ -357,9 +366,9 @@ def _find_path(path):
         parts[2] = origin
         origin = urlunparse(parts)
 
-        return "".join((m.group("markup"), m.group("quote"), origin, m.group("quote")))
+        return "".join((m.group("markup"), m.group("quote"), origin, m.group("quote")))  # type: ignore
```

Comment:
Ah sorry, this was just for getting pyright quiet. It seem to have issues with the return type of `m.group`. Since neither mypy nor pyright are run on the CI I'll remove that statement.